brainglobe_utils/citation/cite.py

```diff
@@ -0,0 +1,300 @@
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Literal
+from warnings import warn
+
+from brainglobe_utils.citation.bibtex_fmt import (
+    BibTexEntry,
+    supported_bibtex_entry_types,
+)
+from brainglobe_utils.citation.repositories import (
+    all_citable_repositories,
+    unique_repositories_from_tools,
+)
+from brainglobe_utils.citation.text_fmt import TextCitation
+
+FORMAT_TO_EXTENSION = {"bibtex": "tex", "text": "txt"}
+EXTENSION_TO_FORMAT = {
+    value: key for key, value in FORMAT_TO_EXTENSION.items()
+}
+
+
+def cite(
+    *tools: str,
+    format: Literal["bibtex", "text"] = "bibtex",
```

Comment:
```suggestion
    format: Literal["bibtex", "text"] = "text",
```
and adjust docstring (and maybe others?) accordingly?
As discussed in https://github.com/brainglobe/brainglobe.github.io/pull/146#discussion_r1476062095